zrml/styx/src/tests.rs

```diff
@@ -0,0 +1,69 @@
+#![cfg(test)]
+
+use crate::{mock::*, Crossings, Error, Event};
+use frame_support::{assert_noop, assert_ok, error::BadOrigin};
+
+#[test]
+fn cross_slashes_funds_and_stores_crossing() {
+    ExtBuilder::default().build().execute_with(|| {
+        frame_system::Pallet::<Runtime>::set_block_number(1);
+        let burn_amount = crate::BurnAmount::<Runtime>::get();
+        let original_balance = Balances::free_balance(&ALICE);
+        assert_ok!(Styx::cross(Origin::signed(ALICE)));
+        let balance_after_crossing = Balances::free_balance(&ALICE);
+        let diff = original_balance - balance_after_crossing;
+        let tx_fee_margin = 5_000_000_000u128;
+        assert_eq!(Crossings::<Runtime>::contains_key(&ALICE), true);
+        assert!(diff >= burn_amount && diff <= burn_amount + tx_fee_margin);
```

Comment:
@maltekliemann Do you have any suggestion for a more accurate way of taking fee into consideration when testing that the amount actually has been slashed by the cross call?

Reply:
Sorry, I forgot to comment on this. There are no fees in the mock. So no need to worry about it. You can just check

```rust
assert_ok!(diff, <Runtime as Config>::BurnValue::get())
```